packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -0,0 +1,145 @@
+import { ICheckStyleProps, ICheckStyles } from './Check.types';
+import {
+  concatStyleSets,
+  getTheme,
+  HighContrastSelector,
+  IStyle,
+  ITheme,
+} from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+
+const DEFAULT_CHECKBOX_HEIGHT: string = '18px';
+
+export const getStyles = (
+  props: ICheckStyleProps
+): ICheckStyles => {
+  const {
+    checkBoxHeight = DEFAULT_CHECKBOX_HEIGHT,
+    checked,
+    className,
+    theme,
+  } = props;
+
+  const { palette, semanticColors } = theme;
+
+  const sharedCircleCheck: IStyle = {
+    fontSize: checkBoxHeight,
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    width: checkBoxHeight,
+    height: checkBoxHeight,
+    textAlign: 'center',
+    verticalAlign: 'middle'
+  };
+
+  return ({
+    root: [
+      'ms-Check',
+
+      {
+        // lineHeight currently needs to be a string to output without 'px'
+        lineHeight: '1',
+        width: checkBoxHeight,
+        height: checkBoxHeight,
+        verticalAlign: 'top',
+        position: 'relative',
+        userSelect: 'none',
+
+        selectors: {
+          ':before': {
+            content: '""',
+            position: 'absolute',
+            top: '1px',
+            right: '1px',
+            bottom: '1px',
+            left: '1px',
+            borderRadius: '50%',
+            opacity: 1,
+            background: semanticColors.bodyBackground,
+          },
+
+          /**
+           * TODO: Come back to this once .checkHost has been
+           * converted to mergeStyles
+           */
+          '.checkHost:hover &, .checkHost:focus &, &:hover, &:focus': {
```

Comment:
Hmmm. I'm not sure this will work. Is "checkHost" a global? Where is this coming from?

Reply:
checkHost is coming from the Check.scss file but only used in DetailsRowCheck.tsx. 

```
<div
  { ...buttonProps }
  role='checkbox'
  className={
    css(className, 'ms-DetailsRow-check', DetailsRowCheckStyles.check, CheckStyles.checkHost, {
      [`ms-DetailsRow-check--isDisabled ${DetailsRowCheckStyles.isDisabled}`]: !props.canSelect,
      [`ms-DetailsRow-check--isHeader ${DetailsRowCheckStyles.isHeader}`]: props.isHeader
    })
  }
  aria-checked={ isPressed }
  data-selection-toggle={ true }
  data-automationid='DetailsRowCheck'
>
  <Check
    checked={ isPressed }
  />
</div>
```